src/ui/server/controllers/insurance/business/contact/validation/rules/first-name.ts

```diff
@@ -0,0 +1,21 @@
+import { ERROR_MESSAGES } from '../../../../../../content-strings';
+import FIELD_IDS from '../../../../../../constants/field-ids/insurance/account';
+import { RequestBody } from '../../../../../../../types';
+import emptyFieldValidation from '../../../../../../shared-validation/empty-field';
+
+const { FIRST_NAME: FIELD_ID } = FIELD_IDS;
+
+const {
+  EXPORTER_BUSINESS: { [FIELD_ID]: ERROR_MESSAGE },
+} = ERROR_MESSAGES.INSURANCE;
+
+/**
+ * validates last name field
+ * checks if response has been provided
+ * @param {RequestBody} responseBody
+ * @param {object} errors
+ * @returns {object} errors
+ */
+const firstName = (responseBody: RequestBody, errors: object) => emptyFieldValidation(responseBody, FIELD_ID, ERROR_MESSAGE.IS_EMPTY, errors);
```

Comment:
Can we extract this into `shared-validation`? Because now it seems we have duplicate validation functions like this for first/last name and email